api/models/Post.js

```diff
@@ -201,11 +213,11 @@ module.exports = bookshelf.Model.extend(Object.assign({
 
     return q
   },
-
+  // TOOD PROPOSAL: check if this can be removed
```

Comment:
i wont flag all of them, but there are more

Reply:
Are there any that stand out to you as still possibly needed? I don't think so but wanted another pair of eyes on them

Reply:
hmm, no im not sure. probably good to get rid of? the socket one might need proposal stuff added to it but would have to be tested